File: sin/sin.java
public class sin {
    // ATOMIC_FUNCTION:  sin(val)
    // TAGS: math，numeric
    // LANG:  java
    // FUNCTION: sin(val)



    public static void main(String args[]) {
        double start_val = Math.PI/2;
        double expected_end_val = 1;

        double end_val = Math.sin(start_val);
        

        // TEST
        if(end_val == expected_end_val) {
            System.out.println("PASS");
        } else {
            System.out.println("FAIL");
        }
    }
}

File: sin/sin.php
<?php
// ATOMIC_FUNCTION:  sin($val)
// TAGS: math，numeric
// LANG:  php
// FUNCTION: sin($val)

$start_val = pi()/2;
$expected_end_val = 1;

$end_val = sin($start_val);

// TEST
if($end_val == $expected_end_val) {
	echo "PASS";
} else {
	
	echo "FAIL";
}

?>
